packages/genesys-spark-components/src/components/beta/gux-rich-text-editor/example.html

```diff
@@ -178,6 +192,60 @@ <h2>No action divider</h2>
   editor.commands.setParagraph();
 })
 
+/* Link action  */
+/* The code below is just an example and can be altered to your preference. */
```

Comment:
There may be betters ways to do this but the dev has full control of how they want it to work.